README.md

```diff
@@ -14,7 +14,7 @@ proofs on blockchain VMs. Users are recommended to check the proof market toolch
 
 - [Boost](https://www.boost.org/) == 1.76.0
 - [cmake](https://cmake.org/) >= 3.5
-- [clang](https://clang.llvm.org/) >= 14.0.6
+- [clang](https://clang.llvm.org/) == 12.0.0
```

Comment:
Do we really need anything related to c++ in this repo?

Reply:
Agreed , removed.